chef/cookbooks/tempest/recipes/config.rb

```diff
@@ -123,6 +123,8 @@
 
 machine_id_file = node[:tempest][:tempest_path] + '/machine.id'
 heat_machine_id_file = node[:tempest][:tempest_path] + '/heat_machine.id'
+insecure = ""
+insecure = "--insecure" if keystone_settings["insecure"]
```

Comment:
You also need to check if glance itself is using insecure certificate (keystone might not be using ssl, for instance)